src/main/resources/springintegration/sample-outbound-flow.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess this isn't really a flow file as it's just creating the bean? Is it worth just creating the bean here https://github.com/ONSdigital/rm-sample-service/blob/master/src/main/java/uk/gov/ons/ctp/response/sample/SampleSvcApplication.java?

Reply:
Just doing like all the others. Also I loathe the annotations based configuration, it just looks plain ugly

Reply:
Fair enough if it's already done like this